build.gradle

```diff
@@ -307,6 +307,7 @@ project(':iceberg-spark') {
     compile project(':iceberg-api')
     compile project(':iceberg-common')
     compile project(':iceberg-core')
+    compile project(':iceberg-data')
```

Comment:
Why is this necessary? It looks like all of the code here is added to Spark.

Reply:
`SparkValueConverter` uses the following classes:

```
import org.apache.iceberg.data.GenericRecord;
import org.apache.iceberg.data.Record;
```